shared/tools/RTGen/src/project/RTGen.Python/Generators/PythonGenerator.cs

```diff
@@ -106,7 +106,7 @@ private string GetPyBind11TypeName(IArgument argument)
                     return "const size_t";
                 else if (argumentName == "Bool")
                     return "const bool";
-                else if (argumentName == "IBaseObject")
+                else if (argumentName == "IBaseObject" || argument.IsEvalValue)
```

Comment:
Just a random thought. Would it be acceptable/desirable to turn this into something along the lines of
```suggestion
                else if (argument.IsPolymorphic)
```
where `IsPolymorphic` would be implemented inside `Argument.cs` and would return exactly that result? (I leave it up to others to suggest a slightly better name.)

@Unsttopabull?